app/uk/gov/hmrc/alcoholdutyaccount/connectors/FinancialDataConnector.scala

```diff
@@ -36,7 +36,9 @@ class FinancialDataConnector @Inject() (config: AppConfig, implicit val httpClie
   override protected val logger: Logger = Logger(this.getClass)
 
   def getFinancialData(
-    appaId: String
+    appaId: String,
+    open: Boolean = true,
+    year: Int = 2024
```

Comment:
Shouldn't default to any value

Reply:
It defaults because year makes no sense if onlyOpenItems is true. So I'd need to change it to an option or pass a nonsense value from the caller. In the case of an option I now face the case where the caller has specified onlyOpenItems false, but hasn't provided a year - so now I need to check and throw or handle other ways = more tests to write for coverage. I'll just pass in a dummy year, but IMO it's just as 'wrong'

Reply:
The problem is should a caller which expects onlyOpenItems (BTA tile) have to provide a dummy year? An Option leaves the issue and another test (= another test) that you provided a year when onlyOpenItems was false. Maybe the best solution here is to overload and both to pass parameters to the common version.